src/assets/styles/styles/custom/_button.scss

```diff
@@ -0,0 +1,18 @@
+.btn {
```

Comment:
@Ehevi dodałem podstawowe style do przycisków, żeby dało się w jakiś sposób odróżnić te przyciski od siebie.
Odnośnie nazewnictwa:
Myślę, że ten styl można nazwać `secondary`

![image](https://user-images.githubusercontent.com/21079319/100532986-9a4b9c00-31ff-11eb-81cf-7487263d71b1.png)

Ten - `primary`

![image](https://user-images.githubusercontent.com/21079319/100532996-b2232000-31ff-11eb-97b3-fba05ebdb179.png)

Ten `outlined`

![image](https://user-images.githubusercontent.com/21079319/100532998-bfd8a580-31ff-11eb-8231-2a4dbfbd49da.png)


Na razie to w nich brakuje trochę rzeczy: odstępy, czcionka, zaorkąglenia, etc.